packages/kotti-ui/source/kotti-field-select/components/ActionIconNext.vue

```diff
@@ -0,0 +1,48 @@
+<template>
+	<div
+		:class="classes"
+		role="button"
+		@mouseenter="hoverOnClearIcon = true"
+		@mouseleave="hoverOnClearIcon = false"
+	>
+		<i class="yoco" @click.stop="handleClear" v-text="icon" />
```

Comment:
we might want to not handleClear if it's not clear icon in the first place - testing and changing this myself if needed

Reply:
Will only execute `handleClear` if the `canClear` condition is met

it doesn't make a difference in the Functional behavior at all, it just prevents an extra execution when the field is empty and the clearIcon is not shown (so an optimization and potentially a guard against future bugs)